docs/guides/designing-the-delta-format.md

```diff
@@ -27,7 +27,7 @@ var content = [
 ];
 ```
 
-If we want to add italics, underline, and other formats, we can add this to the main object, but it is cleaner to separate `text` from all of this so we organize formatting under one field, which we will name `attributes`.
+We can add italics, underline, and other formats to the main object if we want to; but it is cleaner to separate `text` from all of this so we organize formatting under one field, which we will name `attributes`.
```

Comment:
This seems to work even better. It still separates the list from the rest, and it encompasses the other clause quite nicely, methinks.